go/mysql/auth_server.go

```diff
@@ -417,6 +431,14 @@ func (n *mysqlNativePasswordAuthMethod) AuthPluginData() ([]byte, error) {
 	return append(salt, 0), nil
 }
 
+func (n *mysqlNativePasswordAuthMethod) HasCompleteAuthPluginData(authResponse []byte) bool {
+	return len(authResponse) > 0
```

Comment:
All implementations for this method are identical, `len(data) > 0`. Do you think that means this method is superfluous and instead should just be an `if len(data) > 0` inline?